vscode-extension/src/extension.ts

```diff
@@ -10,6 +10,7 @@ import {
   RevealOutputChannelOn
 } from 'vscode-languageclient';
 import { Requirements } from './requirements';
+import { exec } from 'child_process';
```

Comment:
by the way, this and the other import above were added automatically as I was typing ❤️  /cc @olafurpg

Reply:
I've had this for Scala in IJ for a long time, hard to live without once you get used to it :)